app/assets/stylesheets/mo/_utilities.scss

```diff
@@ -114,41 +114,41 @@
 }
 
 .mx-1 {
-  margin-left: 0.25em !important;
-  margin-right: 0.25em !important;
+  margin-left: 0.25rem !important;
+  margin-right: 0.25rem !important;
```

Comment:
Neat.  Are "rem" units new?  Looks like most posts about it are from 2022.

Reply:
Not new. They were the [hot new thing in 2012-ish](https://caniuse.com/?search=rem), but there are other new units turning heads these days. 

These style classes are all cribbed directly from Bootstrap-4, with the idea there would be minimal adjustment once we're on that branch. They're also just so freakin handy.